### DISCUSSION TITLE
ShaderGlass instantly crashes after launching

### DISCUSSION THREAD

**Discussion by lordgeyik** [mausimus/ShaderGlass#75] [General]
ShaderGlass 0.9 instantly crashes after launching on Windows 11.

**Comment by Skyro468** [comment]
Try creating a shortcut and launching it with -p to start it paused. If you are having the same issue as I am having, then it is something to do with whenever it starts trying to capture a window. Starting it paused, you can change the input to a file like a picture and it works fine, or at least for me. But as soon as I swap to a window or display it crashes with no errer. I made an issue with this earlier.

source\repos\ShaderGlass\ShaderGlass\Util\capture.desktop.interop.h
```
Exception thrown at 0x00007FF9755DCF19 in ShaderGlass.exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x000000E8BD7FD720.
Call Stack: >	ShaderGlass.exe!util::CreateCaptureItemForMonitor(HMONITOR__ * hmon) Line 20	C++
```
